tests/python/relay/test_op_level5.py

```diff
@@ -222,6 +222,8 @@ def verify_get_valid_counts(dshape, score_threshold, id_index, score_index):
         func = relay.Function([x], z.astuple())
         func = run_infer_type(func)
         for target, ctx in ctx_list():
+            if target == 'cuda':
```

Comment:
It would be great if we can recover the testcase here

Reply:
I recovered the count test, for the array rearrangement test, since we remove it, I put a comment there.